TypeScript/index.js

```diff
@@ -4,61 +4,59 @@ module.exports = {
     'tslint:recommended',
   ],
   rules: {
-    // 들여쓰기는 공백 2칸을 사용합니다.
+    // 들여쓰기는 공백 2칸을 사용한다.
     'indent': {
       options: [
         'spaces',
         2,
       ],
     },
 
-    // 인터페이스에서 I 접두사를 붙이지 않습니다.
+    // 인터페이스에서 I 접두사를 붙이지 않는다.
```

Comment:
```suggestion
    // 인터페이스 이름에 I 접두사를 붙이지 않는다.
```
이 표현이 더 매끄러운 것 같습니다.